### PR TITLE
feat: added navbar

### PR DESCRIPTION
**Pull request by poojakelkar**
chrome-capture-2022-6-15 (1)

### CODE REVIEW

**Review by chanchal-panpaliya — COMMENTED** [review]
1. Code looks clean 
2. Looks good to merge.
3. Good UI

File: src/component/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+export const Navbar = () => {
+    const { user } = useSelector((store) => store.user);
+    const dispatch = useDispatch();
+    const [logoutbtn, setLogoutbtn] = useState(false);
+    console.log({ user });
+
+    const togglebtn = () => {
+        dispatch(toggleSidebar());
+        console.log("hello");
```
Comment (shlok30) [inline src/component/Navbar.js:16]: Remove console.log before merging

File: src/features/user/userThunk.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        return resp.data;
+    } catch (error) {
+        toast.error(error.response.data.msg);
+        return checkForUnauthorizeResponse(error, thunkAPI);
+    }
+};
+
+export const updateUserThunk = async (url, user, thunkAPI) => {
+    try {
+        console.log(thunkAPI.getState());
```
Comment (shlok30) [inline src/features/user/userThunk.js:29]: Remove console.log

**Review by shlok30 — APPROVED** [review]
1)Clean Code
2)Nice UI
3)Looks good to merge

File: src/component/Navbar.js (hunk 1 above)
Comment (poojakelkar) [reply src/component/Navbar.js:16]: yeah sure thanks

**Pull request merged**